ProjectSidewalk/SidewalkWebpage
# 1271 rename problem_* tables to label_*

NOTE: Do not review this PR until PR #1319 (updating mission infrastructure) has been merged. I branched this code off of the code in that PR instead of the code on develop.

Resolves #1271  

This PR renames the `problem_*` tables to `label_*` to match the other label related columns (like `label_point` and `label_tag`).

Changes:
* Renamed `problem_description` table to `label_description` and renamed its `problem_description_id` column to `label_description_id`.
* Renamed the `problem_severity` table to `label_severity` and renamed its `problem_severity_id` column to `label_severity_id`.
* Renamed the `problem_temporariness` table to `label_temporariness` and renamed its `problem_temporariness_id` column to `label_temporariness_id` and its `temporary_problem` column to `temporary`.
* Modified the relevant code and comments in the front- and back-ends to use this new naming convention.

Testing:
1. Make sure that you can compile and run the code on the `1271-rename-problem-tables` branch.
1. Verify that the tables and columns mentioned above have been renamed appropriately in the database after running the code.
1. Audit a bit with the tool, making sure to test that you can add severity, temporariness, and descriptions to labels, and that these show up in the database. Also test that you can modify the severity/temp/description and that those show up in the database as well.
1. Switch to the `develop` branch, make sure that you can compile and run the code again.
1. Verify that the names of the tables and columns have successfully been reverted back to their original names.
